ci-operator/step-registry/openshift/e2e/libvirt/upi/openshift-e2e-libvirt-upi-workflow.yaml

```diff
@@ -0,0 +1,23 @@
+workflow:
+  as: openshift-e2e-libvirt-upi
+  steps:
+    pre:
+    - ref: upi-libvirt-cleanup-pre
+    - chain: upi-conf-libvirt
+    - ref: ipi-debug-missing-static-pod-controller-degraded
+    - ref: ipi-conf-etcd-on-ramfs
+    - chain: upi-install-libvirt
+    test:
+    - chain: openshift-e2e-libvirt-test
+    post:
+    - chain: gather
+    - ref: upi-libvirt-cleanup-post
```

Comment:
Nit: for future refactor - we don't have a deprovision step for upi. We should probably run destroy cluster and then the libvirt cleanup. I don't know if destroy cluster actually does anything in UPI clusters tho. Something to look into.